src/plugin/plugin.ts

```diff
@@ -454,6 +462,37 @@ export interface PurchasesStoreProduct {
    * Note: Not available for Amazon.
    */
   readonly subscriptionPeriod: string | null;
+
+  /**
+   * Default subscription option for a product. Google Play only.
```

Comment:
Also worth mentioning this will be null for non-subscriptions? Same for `subscriptionOptions`